### PR TITLE
Greg/united way

### PR DESCRIPTION
**Pull request by GodGreg**

### CODE REVIEW

File: backend/routes/admin.js [hunk 1]
```diff
@@ -32,9 +32,7 @@ router.get("/candidate-submissions", function(req, res) {
                      { $and:
                          [
                           { $eq: ["$applicationId", "$$appId"] },
-                          { $ne: ["$Rating", -1]},
-                          { $ne: ["$userId", "vBUgTex5MeNd57fdB8u4wv7kXZ52"]},
-                          { $ne: ["$userId", "hM9QRmlybTdaQkLX25FupXqjiuF2"]}
+                          { $ne: ["$rating", -1]},
```
Comment (brittanylau) [inline backend/routes/admin.js:35]: i'm honoured 😇

File: src/Components/AllCandidates/AllCandidates.jsx [hunk 2]
```diff
@@ -61,12 +61,22 @@ export default class AllCandidates extends Component {
...
+          user.userId !== "hM9QRmlybTdaQkLX25FupXqjiuF2"
+        )
+      });
       this.setState({
         totalReviews: users.length
       });
     });
 
     GET.getCandidateSubmissions().then((data) => {
+      console.log(data);
```
Comment (ahmedhamodi) [inline src/Components/AllCandidates/AllCandidates.jsx:79]: Should this console log be removed?

File: src/Components/AllCandidates/AllCandidates.jsx [hunk 3]
```diff
@@ -128,6 +138,7 @@ export default class AllCandidates extends Component {
 
   render() {
     this.calculateAverageRanking()
+    console.log(this.state.applications)
```
Comment (ahmedhamodi) [inline src/Components/AllCandidates/AllCandidates.jsx:141]: Should this be removed too?

File: src/Components/Categories/Categories.jsx [hunk 4]
```diff
@@ -85,7 +85,11 @@ const useStyles = makeStyles({
 });
 
 const Categories = ({ categoryData }) => {
-  const { contact } = categoryData;
+
+  console.log(categoryData)
```
Comment (ahmedhamodi) [inline src/Components/Categories/Categories.jsx:89]: Unneeded console log

File: src/Components/DecisionCanvas/DecisionCanvas.jsx [hunk 5]
```diff
@@ -84,6 +84,8 @@ function DecisionCanvas({ update, review, categoryData }) {
     }, {});
   }, [review]);
 
+  let a = 0;
+  let b = 0;
```
Comment (ahmedhamodi) [inline src/Components/DecisionCanvas/DecisionCanvas.jsx:88]: Are these variables needed?

File: src/Components/Categories/Categories.jsx (hunk 4 above)
Comment (GodGreg) [reply src/Components/Categories/Categories.jsx:89]: yeah not needed

File: src/Components/DecisionCanvas/DecisionCanvas.jsx (hunk 5 above)
Comment (GodGreg) [reply src/Components/DecisionCanvas/DecisionCanvas.jsx:88]: yeah i don't know what these are for lol. good catch

**Pull request merged**